libs/langgraph/langgraph/prebuilt/tool_node.py

```diff
@@ -214,6 +215,20 @@ def _func(
         config_list = get_config_list(config, len(tool_calls))
         with get_executor_for_config(config) as executor:
             outputs = [*executor.map(self._run_one, tool_calls, config_list)]
+
+        commands: list[Command] = [
+            output for output in outputs if isinstance(output, Command)
+        ]
+        if len(commands) > 0:
+            if len(commands) != len(outputs):
+                raise ValueError(
+                    f"Cannot mix Command and non-command (message) tool outputs, got the following outputs: {outputs}"
+                )
+
+            # Users that want to include ToolMessages in the state update
```

Comment:
this comment doesn't make sense imo. users that don't do this can never use this thread again as then all future llm calls will fail

Reply:
that's true for using within the same graph, but you might also want to jump to a different node inside the parent graph without any state updates (i.e. navigate to a different agent) without adding tool message. so not sure if we should automatically add tool messages / enforce this somehow

do you have any ideas?

Reply:
resolved this offline -- we'll raise an error if a tool message is not provided in `Command.update`